### PR TITLE
New Adjectives Added

### LINKED ISSUE #4: Add More Adjectives!
_This will be a running issue to let people know how to add adjectives to `words_dir/tweet_adjectives.py`_

---
In the `words_dir/tweet_adjectives.py` file, there are some adjectives separated by comments to have different categories.

---

Within categories, please keep in alphabetical order and all lowercase.

All adjectives added should look like the below
```python
'adjective goes here',
```

If you're adding a new category of items, do it like this
```python
# category name here
'adjective goes here',
'adjective goes here',
'adjective goes here',
```

### Proposed workflow:
1. Fork this repo
2. Add answers to `words_dir/tweet_adjectives.py` keeping to the existing style/conventions:
   * all lower case letters, unless it's a Proper noun
   * follow the examples above
3. Once you are done and checked for typos, commit it & push your work to your fork
4. Submit a PR against this repo

Someone will look it over and, if deemed appropriate, will merge your PR.

Thanks for contributing!

---
*Don't forget to follow the bot on [Twitter](https://www.twitter.com/LGBTQotd) to see the hard work you put in.*

### PR DESCRIPTION
**Pull request by Arijit-101**
Fixes #4

### CODE REVIEW

**Comment by Arijit-101** [comment]
It said Python Linter failed.. why is it occuring ? and does that mean that the pull request failed

File: words_dir/tweet_adjectives.py [hunk 1]
```diff
@@ -9,11 +9,14 @@
 # NOTE: must fit in a tweet, so not longer than this line -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 
 adjectives = [
+    'aback',
+    'abaft'
```
Comment (VickiLanger) [inline words_dir/tweet_adjectives.py]: Would you mind adding a comma to the end of this line?

File: words_dir/tweet_adjectives.py [hunk 2]
```diff
@@ -453,14 +456,23 @@
...
     'irritable',
+    'likeable',
     'mischievous',
+    'polite',
     'sagacious',
     'stoic',
     'strict',
-    'strong'
+    'strong',
+    'understanding'
```
Comment (VickiLanger) [inline words_dir/tweet_adjectives.py]: Would you mind adding a comma to the end of this line?

File: words_dir/tweet_adjectives.py [hunk 3]
```diff
@@ -453,14 +456,23 @@
...
+    'weak'
```
Comment (VickiLanger) [inline words_dir/tweet_adjectives.py:477]: Would you mind adding a comma to the end of this line?

**Review by VickiLanger — CHANGES_REQUESTED** [review]
Awesome! Thanks for all of these additions and fixes. For a couple of them, would you mind adding commas to the end of the lines? Without them, they'll get smashed together with the next line.

**Comment by VickiLanger** [comment]
> It said Python Linter failed.. why is it occuring ? and does that mean that the pull request failed

Good question. The liner is a bit messed up right now. It's actually catching on some other file no one touches

File: words_dir/tweet_adjectives.py (hunk 3 above)
Comment (Arijit-101) [reply words_dir/tweet_adjectives.py:477]: Done

**Comment by Arijit-101** [comment]
The python linter failed again !

**Comment by bobsany16** [comment]
> The python linter failed again !

Yeh, the python linter failed because when I implemented it, I could not figure out how to stop telling it to reformat the files (as it is python Black linter's job) so that's why it's failing. So far, I don't know how to fix that yet 😅

**Review by VickiLanger — APPROVED** [review]
Thanks for your contributions!

---
*Don't forget to follow the bot on [Twitter](https://www.twitter.com/LGBTQotd) to see the hard work you put in.*

**Pull request merged**